.gitignore

```diff
@@ -926,3 +932,4 @@ lib/cpluff/stamp-h1
 /addons/library.xbmc.pvr/libXBMC_pvr.lib
 
 /pvr-addons
+/adsp-addons
```

Comment:
I remove the /adsp-addons there, has forgot this.

There is still for /pvr-addons a:
ifneq ($(wildcard pvr-addons/Makefile.am),)
BOOTSTRAP_SUBDIRS += pvr-addons/configure.ac
endif
inside bootstrap.mk, also remove this?